src/main/scala/where/Condition.scala

```diff
@@ -76,3 +77,12 @@ case class Expression(string: String, references: Reference*) extends Condition
   def identifiables: Set[Identifiable] =
     (references flatMap (_.getReferenceable)).toSet
 }
+
+case class NoRelationshipCondition(node: Node) extends Condition {
```

Comment:
is there a chance we could get away with not needing to redundantly pass in `node` here? `node where (HasNoRelationships withLabel "labelName") returns startNode` would read nicely. I don't think the information is available to actually do that though..